lib/graphql_devise.rb

```diff
@@ -3,6 +3,25 @@
 require 'rails'
 require 'graphql'
 require 'devise_token_auth'
+require 'zeitwerk'
+require_relative '../app/models/graphql_devise/concerns/model'
```

Comment:
If we are going to explicitly require these, we should probably move them to lib so they are not autoloaded by the rails engine

Reply:
I was planning to add new concerns that comply with Zeitwerk name constraints inside of lib and add deprecation warnings to the current constraints (for instance GraphqlDevise::Authenticable vs GraphqlDevise::Concerns::Model). Let me know if you still consider we should move them and I can address it in the next commits.